ramls/patron-notice-policy.raml

```diff
@@ -82,6 +82,11 @@ traits:
         responses:
           204:
             description: "Source record deleted"
+          422:
```

Comment:
How was it decided that the response would be a 422/JSON rather than a 400/plain text (as is [already defined for deletion](https://github.com/folio-org/raml/blob/69f6074f5f48fd58e379e9ae76c892aa68c115ea/rtypes/item-collection.raml#L41))

As this is a new status code, is the client prepared to handle a 422 response?

Reply:
Actually client is not prepared yet. As 400 is already defined for constraint violation during deletion I will change status to 400. Thank you for the link.

Reply:
done